x-pack/solutions/observability/packages/kbn-scout-oblt/src/playwright/fixtures/parallel_run_fixtures.ts

```diff
@@ -46,3 +44,8 @@ export const spaceTest = spaceBase.extend<ObltParallelTestFixtures, ObltParallel
     { scope: 'worker' },
   ],
 });
+
+/**
+ * Should be used test spec files, running in parallel in isolated spaces against the same Kibana instance.
+ */
+export const spaceTest = mergeTests(spaceFixture, sloDataFixture, annotationsDataFixture);
```

Comment:
All `spaceTest` fixtures should be supporting Kibana spaces or at least don't affect other parallel tests (note: Scout parallel tests are run against the single Kibana and ES instances but each test thread in its own Kibana Space)

I don't know much about `sloDataFixture` or `annotationsDataFixture`, but I don't see them utilising Kibana Spaces. If it won't work in parallel, please remove it and keep only for "sequential" run with `test`